app/services/MemberDetailsUploadValidator.scala

```diff
@@ -42,7 +42,7 @@ class MemberDetailsUploadValidator @Inject()(
   textFormProvider: TextFormProvider
 )(implicit ec: ExecutionContext) {
 
-  private val memberDetailsForm = MemberDetailsController.form(nameDOBFormProvider)
+  private val memberDetailsForm = MemberDetailsController.form(nameDOBFormProvider, None)
```

Comment:
On line https://github.com/hmrc/pension-scheme-return-frontend/pull/256/files#diff-f30e455ec087fdbbe6a8a5023b69278b6159628debc93cca75c943fe6f488eeeR110 there is a GetOrElse to change this "None" with a LocalDate.now